iFixit/react-commerce
# Fix price filter bugs

This PR closes #129 and #130

## QA

Until we setup the staging servers we need to clone the repo and test it locally.

1. Clone this repo
2. Make sure you are running on node v14 (if you are using nvm run `nvm use` in the project working directory)
3. Install dependencies `npm install`
4. Rename `backend/.env.example` to `backend/.env` (this will enable db seed)
5. Run dev server: `npm run dev`
6. Make sure that bug described by #129 is fixed
7. Make sure that bug described by #130 is fixed

### Access to local strapi instance

Email: `strapi@ifixit.com`
Password: `Password1`
